### PR TITLE
[CELEBORN-547] Refactor request related API

### PR DESCRIPTION
**Pull request by AngersZhuuuu**
### What changes were proposed in this pull request?
Refactor request related API

### CODE REVIEW

File: client/src/main/scala/org/apache/celeborn/client/LifecycleManager.scala [hunk 1]
```diff
@@ -1059,7 +1053,7 @@ class LifecycleManager(appId: String, val conf: CelebornConf) extends RpcEndpoin
     }
   }
 
-  private def requestDestroy(endpoint: RpcEndpointRef, message: Destroy): DestroyResponse = {
+  private def requestWorkerDestroy(endpoint: RpcEndpointRef, message: Destroy): DestroyResponse = {
```
Comment (FMX) [inline client/src/main/scala/org/apache/celeborn/client/LifecycleManager.scala:1056]: I think it should be renamed to 'requestWorkerDestroySlots' and the Destroy message should also be renamed.
The name 'WorkerDestroy' is misleading.

Comment (AngersZhuuuu) [reply client/src/main/scala/org/apache/celeborn/client/LifecycleManager.scala:1056]: Year, should do like this. Let me do this first.

Comment (FMX) [reply client/src/main/scala/org/apache/celeborn/client/LifecycleManager.scala:1056]: OK.

**Review by FMX — APPROVED** [review]
LGTM.

**Pull request merged**